modules/ROOT/pages/scopes-and-collections-config.adoc

```diff
@@ -0,0 +1,191 @@
+
+The default is unique to the bucket creation process and predominantly serves the purpose of backward compatibility.
+
+Please be aware that only Couchbase Server (CBS) allows using the word 'default' in naming conventions. 
+This explains why you may see the term _default.default when creating a new bucket. 
+Couchbase encourage you to explore and experiment with this concept within your Couchbase instance for practical experience.
+
+== Configuration Options
+
+You have these options to configure Scopes and Collections Sync for Gateway:
+
+* *custom scope/custom collection (the option we cover here)* +
+Gain maximum flexibility in organizing and accessing your data by customizing it to suit your needs.
+
+[NOTE]
+====
+Scopes is not required for upgrading to Sync Gateway 3.1. 
+If it is not specified in the configuration, the previous versions are still compatible with the new version. Therefore, the provided examples are not necessary for upgrading to the latest version of Sync Gateway.
+====
+
+* *default scope/custom collection* +
+Create a custom collection with the default scope can lead to efficient data organization and quicker access control management.
+
+* *default scope/default collection* +
+Work within the default Scopes and Collections. This makes managing data easy and simple without creating extra scopes or collections.
+
+[NOTE]
+====
+Importing a dataset created in previous versions of Couchbase Server, the documents will now be saved automatically in the "_default" collection and scope.
+====
+
+== Configuration Example: _custom scope/_custom collection
+
+In this example, we cover a configuration with a custom Scope and 2 custom Collections:
+
+[source,javascript]
+----
+{
+  "name": "db",
+  "bucket": "bucket",
+  "scopes" : {
+    "scope1": {
+      "collections": {
+        "collection1" : {
+            "sync": `
+                function(doc, oldDoc, meta) {
+                    if (doc.type != "default") {
+                        throw({forbidden : "Rejected document"})
+                    }
+                    channel("legacy")
+                }
+            `,
+            "import_filter": `
+              function(doc) {
+                return doc.type == "mobile"
+              }
+            `
+        },
+        "collection2" : {
+          "sync": `
+            function(doc, oldDoc, meta) {
+              channel("collection1")
+            }
+          `,
+          "import_filter": `
+              function(doc) {
+                return doc.type == "mobile"
+              }
+          `
+        }
+      }
+    }
+  },
+  "num_index_replicas": 0
+}
+----
+
+== Collections Configuration Description
+
+Here is the description for the custom `Scope` with 2 custom `Collections` configuration. 
```

Comment:
(Avoid starting a section with Here is, here are)

This section describes the custom scope